.codecov.yml

```diff
@@ -10,20 +10,33 @@ coverage:
   range: "70...100"
   status:
     project:
+      default: false
       bot:
-        paths: ["modmail/"]
+        paths:
+          - modmail
         # basic
         target: auto
-        threshold: 1%
+        threshold: 5%
         base: auto
         flags:
           - unit
        # advanced settings
         branches:
           - main
+        if_ci_failed: ignore #success, failure, error, ignore
+        informational: true
+        only_pulls: true
```

Comment:
Also what does only pulls do here? Should that be commented or am I just ignorant on how this config works.

Reply:
Its the available options, since the docs example has those comments, but also does not list them as possible options later.

Reply:
Sorry, only replied to the last part of your comment.

"only_pulls" makes codecov not run on commits that aren't part of a pull yet.